### ISSUE TITLE
Clarify "One path to production"

### ISSUE THREAD

**Issue by bdfinst** [Minimum-CD/cd-manifesto#32]
Mike Nygard was questioning this practice. When I clarified that it meant that the only path to production was the pipeline and not that there was only one pipeline for the entire organization, he was happier. Suggest we change it to "The pipeline is the only path to production"

**Comment by patrickkelso** [comment]
"A pipeline is the only path to production"?

**Comment by patrickkelso** [comment]
I agree. I think it goes to making sure we know what "the pipeline" refers to.

- The application pipeline ...
- An automated pipeline ...

**Comment by bdfinst** [comment]
Could link to Dave's "Anatomy of a Pipeline" video.
